prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -1414,10 +1415,7 @@ class DatabaseAccess(val create: DSLContext) : Logging {
             config.addDataSourceProperty("cachePrepStmts", "true")
             config.addDataSourceProperty("prepStmtCacheSize", "250")
             config.addDataSourceProperty("prepStmtCacheSqlLimit", "2048")
-            config.addDataSourceProperty(
```

Comment:
This code was setting connection timeout to 60sec but in staging and prod the effective timeout was 30 seconds. So, I deleted this chunk of code and set the connection timeout via the connectionTimeout property, hoping this will fix the issue.

See the incident report linked in the PR description for more details.